src/components/Slide/Slide.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as Styled from './Slide.styled';
+import PropTypes from 'prop-types';
+
+const Slide = ({ banner }) => {
+  const {
+    data: { title, description, main_image },
```

Comment:
These derived props are implementation detail of the json structure, you may just expect `description`, `title` and `imgSource` props in this component. This will avoid refactoring if the json response structure changes.